plugins/adr/src/components/EntityAdrContent/EntityAdrContent.tsx

```diff
@@ -68,9 +68,10 @@ const useStyles = makeStyles((theme: Theme) => ({
     color: theme.palette.grey[700],
     marginBottom: theme.spacing(1),
   },
-  adrChip: {
-    position: 'absolute',
-    right: 0,
+  adrBox: {
+    display: 'flex',
+    justifyContent: 'space-around',
```

Comment:
Minor thing here, but I think `space-between` might look nicer as it will push both the chip and the date flush to the padding like the text.

Reply:
@npiyush97 I played with this locally and I think @benjdlambert's suggestion is the correct way to style this nicely here - what do you think?

Reply:
Dont know may be because of the different screen sizes to me space-between the chip and date are not aligned,but space-around does.